torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1751,6 +1772,7 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            supports_tensor_out=True),
     UnaryUfuncInfo('neg',
                    ref=np.negative,
+                   aliases=('negative', ),
```

Comment:
See https://github.com/pytorch/pytorch/pull/52037/files, which removes the AliasInfos for negative. This PR also has to remove the AliasInfos for the other aliases it's adding.

Reply:
Thanks for the heads up. Will remove the alias entries for now.